src/main/java/StringCalculatorStep8.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+
+public class StringCalculatorStep8 {
+    public static final int Add(final String numbers) {
+        int returnValue = 0;
+        String[] numbersArray = numbers.split(",");
+        if (numbersArray.length > 2) {
+            throw new RuntimeException("Up to 2 numbers separated by comma (,) are allowed");
+
+        } else {
+            if (!numbers.isEmpty()) {
+                returnValue = Arrays.stream(numbersArray).mapToInt(Integer::parseInt).sum();
+            }
+        }
+        return returnValue;
```

Comment:
Consider using logger or proper error handling instead of printing to System.out directly.